dashboard/app/controllers/reference_guides_controller.rb

```diff
@@ -0,0 +1,15 @@
+class ReferenceGuidesController < ApplicationController
+  include CurriculumHelper
+  authorize_resource
+
+  before_action :require_levelbuilder_mode_or_test_env
+
+  def show
+    if params[:course_course_name] && params[:key]
```

Comment:
It seems like it might be easier to manage this code going forward if we start the controller with 
`render :not_found if params[:course_course_name] && params[:key]` that way everything does not have to be inside of an if statement.